### ISSUE TITLE
Training yolo doesn't improve the accuracy

### ISSUE THREAD

**Issue by androuino** [dmlc/gluon-cv#1469]
Hi, I am training the yolov3 with darken53_custom backbone but the classes and the mAP stay at 0.0 and doesn't look to improve because my epochs went to 50 but the value remained 0.0, all of them.

Am running with macOS CPU, my learning rate is 0.001 and my batches is set to 2. Then my images input size is 800x1144 then resizing it to 608 with hyperparameters. My dataset is in voc format, 7 images for training, and 3 images for validation. This is just for testing training yolov3 on my computer.

Could someone tell me what's wrong with my settings or my dataset?

Thank you in advance.

**Comment by Aktcob** [comment]
u can test mAP of training dataset, to make sure that no problem with the training process

**Comment by zhreshold** [comment]
It's unlikely that you can successfully train with <10 images, let along you are using cpu. You can refer to the fine-tuning guide that allows you to reuse pre-trained yolo detectors
